### ISSUE TITLE
Error on windows install

### ISSUE THREAD

**Issue by jotakar** [silexlabs/Silex#811]
After updated versions
node 8.12
npm 6.4
Python 3.7

Indeed I can not install  in a Windows enviroment. Now errors are

`gyp ERR! configure error
gyp ERR! stack Error: Command failed: C:\website\server\python\python.EXE -c import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack   File "<string>", line 1
gyp ERR! stack     import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack                                ^
gyp ERR! stack SyntaxError: invalid syntax
`
And a few lines down:
`
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! iconv@2.3.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the iconv@2.3.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
`
Bye

**Comment by lexoyo** [comment]
Windows... It's a server so it is not well rested on Windows

The solutions are simple
* You help me debug on Windows and I'll be happy but it will take months
* You help me build a Windows version of @JbIPS desktop app (https://github.com/silexlabs/Silex/releases) 
* You use the docket version which was broken since a week or two, help us here https://github.com/silexlabs/Silex/issues/699
* You use Silex online on https://editor.silex.me
* You switch to Linux ;)

**Comment by JbIPS** [comment]
Hello ! What's your Python version?

**Comment by jotakar** [comment]
Well I finally installed on a linux server, but now my problem is localhost.
I access this server via net and after install I can see the first page but any templata I want to use look for images an resources in localhost, but this server is 192.168.1.3 (localhost is my main desktop: Windows)
So what can I do to get the app installed as a web in a IP not a localhost?
Thanks

**Comment by jotakar** [comment]
Eureka!!!
My experience as a help for other users:
After install using **npm install** and **npm build**, I get errors because some files had not been copied into dist  folders. These files are those in folder: templates and  wysihtml  
I noticed it because of errors messages in developer panel of browser.
I see in **package.json** a buid script **postinstall** that looks like it copies all these files to their places 
So you must do a **npm run postinstall** (or modify build script inpackage.json before install)
Last file that is not in its place is debug.js, I copied it by hand.
Now it's working, but I have to test it more time.
Now my task is to understand how the app work. This will be my first time with node.js!!
